src/routes/blog/post/announcing-csv-imports/+page.markdoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Announcing: Document imports from CSV files"
+description: Learn how to import documents into your Appwrite collections using a simple CSV file — a new feature built on top of our powerful migration APIs.
+date: 2025-04-20
+# update this cover later, once available!
+cover: /images/blog/planetscale-databases-alternative/cover.png
+timeToRead: 5
+author: darshan-pandya
+category: announcement
+featured: false
+---
+
+We're introducing a new way to populate your Appwrite databases: **document imports from CSV files**.
+
+Built on top of Appwrite's powerful migration APIs, this feature makes it easy to bring in large datasets, seed collections, or migrate structured data—using nothing more than a CSV file.
+
+## How it works
+
+To get started, create a collection and define its attributes in the Appwrite Console. Your CSV file should follow a standard format:
+
+- The first row must be a header containing attribute names that match your collection
+- Each subsequent row represents a document, with values separated by commas
+- You can include the `$id` column to assign custom document IDs
+
+All required attributes must be present in the CSV, and Appwrite will validate each row before importing it.
+
+{% info title="Note" %}
+The selected file must be stored in a bucket that is not encrypted and does not have compression enabled.
```

Comment:
Didn't we discuss compression and encryption should not be a limitation for this feature? Why are we still limited?
Both compression and encryption in Appwrite storage are limited in size so we never reach a situation where we need to load a risky file size in full to memory. Please lets make sure this is not a product limitation.

Reply:
I think we finalized that this should be like this because then a file could be huge and therefore loading it into memory can cause issues for decryption and decompression.

Compression might not be a big issue for smaller files, but the buckets are encrypted by default -
https://github.com/appwrite/appwrite/pull/9622#discussion_r2041833323